.github/ISSUE_TEMPLATE/new_podcast.yml

```diff
@@ -0,0 +1,71 @@
+name: New podcast
+description: Fill this form to add your podcast to the website.
+title: "[New][Podcast]: "
+labels: ["new", "podcast"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Tell us about your blog.
+
+  - type: input
+    id: author
```

Comment:
Could be `host` instead, the issue creator is someone we will know by default, but sometimes contributors may want to add a podcast which is hosted by a different group of people. Podcasts as I mentioned could be hosted by one or many persons, so I strongly believe that having a placeholder asking for multiple values, comma separated, will be great (p.s. this is an open topic, just sharing my thoughts)